packages/@o3r/pipeline/README.md

```diff
@@ -0,0 +1,44 @@
+<h1 align="center">Otter Pipeline</h1>
+<p align="center">
+  <img src="https://raw.githubusercontent.com/AmadeusITGroup/otter/main/assets/logo/otter.png" alt="Super cute Otter!" width="40%"/>
+</p>
+
+This package is an [Otter Framework Module](https://github.com/AmadeusITGroup/otter/tree/main/docs/core/MODULE.md).
+<br />
+<br />
+
+## Description
+
+[![Stable Version](https://img.shields.io/npm/v/@o3r/pipeline?style=for-the-badge)](https://www.npmjs.com/package/@o3r/pipeline)
+[![Bundle Size](https://img.shields.io/bundlephobia/min/@o3r/pipeline?color=green&style=for-the-badge)](https://www.npmjs.com/package/@o3r/pipeline)
+
+This module contains tooling around DevOps toolchains.
+
+## How to install
+
+```shell
+ng add @o3r/pipeline
+```
+
+## Description
+
+The `ng add` schematic for Otter Pipeline helps you set up a DevOps pipeline for your frontend project. This schematic configures the necessary CI runner and npm registry settings to streamline your development workflow.
+
+### Properties
+
+- `toolchain`: The DevOps toolchain to create. For now, only `GitHub` is supported.
+- `runner`: The CI runner. Default is `ubuntu-latest`.
+- `npmRegistry`: A private npm registry. By default, the public one will be used.
+
+### Usage
+
+Here is an example of how to use the `ng add` schematics with parameters:
+
+```shell
+ng add @o3r/pipeline --runner=windows-latest --npmRegistry=https://custom-registry.example.com
```

Comment:
I think you should add the code snippet of the "standard" usage (without the options) for copy/past purpose before writing this example

Reply:
It is the case on line 20: `ng add @o3r/pipeline`